birdhouse/config/jupyterhub/default.env

```diff
@@ -5,7 +5,7 @@
 # are applied and must be added to the list of DELAYED_EVAL.
 
 export JUPYTERHUB_DOCKER=pavics/jupyterhub
-export JUPYTERHUB_VERSION=4.0.2-20231002
+export JUPYTERHUB_VERSION=4.0.2-20231024
```

Comment:
I am guessing the `JUPYTERHUB_VERSION` will change again after https://github.com/Ouranosinc/jupyterhub/pull/23 is merged an a new build is triggered on DockerHub?

Reply:
Yeah, I'll update it shortly.